qiskit/transpiler/__init__.py

```diff
@@ -509,466 +1167,19 @@
-the target IBM Quantum device (the :class:`~.GenericBackendV2` class generates
-a fake backend with a specified number of qubits for test purposes):
-
-.. plot::
-   :alt: Circuit diagram output by the previous code.
-   :include-source:
-   :context: reset
-
-   from qiskit import transpile
-   from qiskit import QuantumCircuit
-   from qiskit.providers.fake_provider import GenericBackendV2
-
-   backend = GenericBackendV2(5)
-
-   qc = QuantumCircuit(2, 1)
-
-   qc.h(0)
-   qc.x(1)
-   qc.cp(np.pi/4, 0, 1)
-   qc.h(0)
-   qc.measure([0], [0])
-
-   qc_basis = transpile(qc, backend)
-   qc_basis.draw(output='mpl')
-
-A few things to highlight. First, the circuit has gotten longer with respect to the
-original.  This can be verified by checking the depth of both circuits:
-
-.. plot::
-   :include-source:
-   :nofigs:
-   :context:
-
-   print('Original depth:', qc.depth(), 'Decomposed Depth:', qc_basis.depth())
-
-.. code-block:: text
-
-    Original depth: 4 Decomposed Depth: 10
-
-Second, although we had a single controlled gate, the fact that it was not in the basis
-set means that, when expanded, it requires more than a single :class:`~.CXGate` to implement.
-All said, unrolling to the basis set of gates leads to an increase in the depth of a
-quantum circuit and the number of gates.
-
-It is important to highlight two special cases:
-
-1. If A swap gate is not a native gate and must be decomposed this requires three CNOT gates:
-
-   .. plot::
-      :include-source:
-      :nofigs:
-
-      from qiskit.providers.fake_provider import GenericBackendV2
-      backend = GenericBackendV2(5)
-
-      print(backend.operation_names)
-
-   .. code-block:: text
-
-      ['id', 'rz', 'sx', 'x', 'cx', 'measure', 'delay']
-
-   .. plot:
-      :alt: Circuit diagram output by the previous code.
-      :include-source:
-
-      from qiskit.circuit import QuantumCircuit
-
-      swap_circ = QuantumCircuit(2)
-      swap_circ.swap(0, 1)
-      swap_circ.decompose().draw(output='mpl')
-
-   As a product of three CNOT gates, swap gates are expensive operations to perform on
-   noisy quantum devices.  However, such operations are usually necessary for embedding a
-   circuit into the limited gate connectivities of many devices. Thus,
-   minimizing the number of swap gates in a circuit is a primary goal in the
-   transpilation process.
 
+.. _transpiler-scheduling-description:
 
-2. A Toffoli, or controlled-controlled-not gate (``ccx``), is a three-qubit gate.  Given
-   that our basis gate set includes only single- and two-qubit gates, it is obvious that
-   this gate must be decomposed.  This decomposition is quite costly:
+Scheduling of circuits
+======================
 
-   .. plot::
-      :alt: Circuit diagram output by the previous code.
-      :include-source:
+..
+    This section is still here because the content hasn't fully migrated to other places yet, unlike
+    other discussions of the components of quantum compilation.
```

Comment:
I was just going to comment on this (I was looking at the rendered version), good clarification.